docs/api.md

```diff
@@ -14,6 +14,10 @@ nav_order: 3
 
 Returns the current config.
 
+### `.identifier` → [String]
```

Comment:
I believe this whole file is auto-generated from code comments (and really shouldn't even be in source control on this branch). Mind making the changes in the comments instead?

Reply:
Sure, I'll drop the changes to this file from this branch. Looks as though we do still need to make the changes to the docs generator for HTML safety, so I'll extract those out to another PR.

I think this might've been a case of old habits – while I'm in here I'll check that our documentation for testing changes to viewcomponent.org are up to date and in a good place.